perf/benchmark/configs/istio/security_authz/latency.yaml

```diff
@@ -0,0 +1,18 @@
+# Data: latency
+# Config: security_authz
+telemetry_mode: "none"
+conn:
+    - 2
+qps:
+    - 1000
+duration: 240
+perf_record: true
+# we only care about both sidecar mode, which is the base line for mTLs overhead comparison.
```

Comment:
the comment mentions mTLS but this test is for authz right?

Reply:
removed~